static/js/login_or_signup.js

```diff
@@ -125,7 +125,13 @@ $(function() {
             if (request.status === 201) {
                 alert('Successfully created user');
             } else {
+                alert('Unknown successful status');
```

Comment:
Is all this alerting temporary? If so consider making a task to take care of it.

Reply:
I mean, all the alerts are temporary. But yeah I'm not sure what the right thing to do is here. Perhaps we need to create a general "something went wrong with that action" popup or something? This case is especially weird, because it would mean the server returned a 2XX code, but one that was unexpected. Like wtf.

Reply:
Ya too much for this code review but maybe create a task to add cool little banner notifications that fade on certain actions. Something kinda like [this](https://css-tricks.com/pop-from-top-notification/)

Reply:
created https://trello.com/c/8VCLI26b